README.md

```diff
@@ -90,12 +90,15 @@ $ cd cuML
 $ mkdir build
 $ cd build
 $ cmake ..
-$ make -j
+$ make -j 
+$ # Instead of previous step if you like you can use cmake directly
+$ cmake --build . --config Release -- -j
```

Comment:
Any advantage to having this alternative way of building using cmake? I'm not against it, but I wonder what is the advantage over just `make -j`

Reply:
It can be launched on Windows if under the hood build system which is not compatible with GNU Make for example - Microsoft Visual Studio / MSVC.

Reply:
That is a great reason :)